tests/server/test_server.py

```diff
@@ -1205,6 +1205,11 @@ async def test_solve_double_subclass():
         output["message"],
     )
 
+    # Check that the knowledge graph node has the correct category
+    assert set(output["message"]["knowledge_graph"]["nodes"]["MONDO:1"]["category"]) == \
```

Comment:
At some point it might be nice to check this as part of `validate_message`.

Reply:
I'm still trying to figure out how that would fit in to the format of the input to validate_message. If you have any ideas let me know.

Reply:
What if we used the same format for the knowledge_graph here as we do for the translator_overlay's kp_data?